recipes/jalview/meta.yaml

```diff
@@ -0,0 +1,32 @@
+
+about:
+    home: http://www.jalview.org/
+    license: GNU GENERAL PUBLIC LICENSE, Version 3.0
+    summary: |
+        Jalview is a free program for multiple sequence alignment editing, visualisation and analysis. 
+        Use it to view and edit sequence alignments, analyse them with phylogenetic trees and principal 
+        components analysis (PCA) plots and explore molecular structures and annotation.
+package:
+    name: jalview
+    version: 2.10.2b2
```

Comment:
You could use template-vars for the number, would make updates easier!

Reply:
problem is, the link uses underscores instead of points... any idea?